### ISSUE TITLE
FR: Transmit Disable

### ISSUE THREAD

**Issue by nayrnet** [wb2osz/direwolf#320]
Story: As an operator, I desire the ability to disable transmit on a station without stopping/restarting/reconfiguring the TNC, effectively turning it into receive-only mode on the fly.

Uses:
- Safe Station Maintenance, Tower Maintenance keeping Rx alive.
- Station Troubleshooting/Testing.
- Scheduled Quiet Times for Interference Reduction.
- High Availability Hotspare, a wrapper could negotiate quorum in a HA setup allowing a single Tx and mult-read setup that can reconfigure itself on the fly.

Proposed Solutions:
 - Existence of a file is checked before keying up and if it exists a message is logged, potentially suppressible. I believe this would be platform-agnostic and backward compatible.

**Comment by wb2osz** [comment]
I agree that it would be useful to change some of the operating parameters on the fly, rather than having to restart.
There is already a transmit inhibit capability (User Guide section 9.2.12) but it is currently only for Linux.
Constantly checking for the presence of a file (more generally a different file for each radio channel), would not scale up well when adding more and more functions to change or query the internal operating state.   As an example, direwolf maintains a list of recently heard stations but there is not a good way for an external application to grab a copy.
This might be a good time to start thinking about a more general, extensible, method for external applications to interact in ways other than transmitting and receiving packets.
